src/util/type.cpp

```diff
@@ -1,27 +1,38 @@
 /*******************************************************************\
 
-Module:
+Module: Data structure for a type of an expression
 
 Author: Daniel Kroening, kroening@kroening.com
+        Maria Svorenova, maria.svorenova@diffblue.com
 
 \*******************************************************************/
 
+/// \file
+/// Data structure for a type of an expression
```

Comment:
This comment feels like it belongs to `typet` rather than this file

Reply:
There's not much more than `typet` in this file. Would you rather remove the file description then? Or should it be something like `Definition of typet class`?

Reply:
I'm not really sure I understand the value of having both a file and a class summary when a file contains exactly one class of the same name. I'd expect to find the documentation on the `typet` so I'd either remove this (preferable) or put just have `typet`?

Reply:
The `\file` documentation appears in the right-hand column at http://cprover.diffblue.com/files.html (just expand some of the folders on the left to see them), which can be a useful overview even if there is more detailed class-level documentation as well?